kuwala/core/canvas/src/components/Nodes/PostgresDataSourceNode.js

```diff
@@ -0,0 +1,111 @@
+import React, {useState} from 'react';
+
+import { Handle } from 'react-flow-renderer';
+import NodeConfigModal from "../Modals/NodeConfigModal";
+import {useStoreActions, useStoreState} from "easy-peasy";
+import PostgreSVG from "../../icons/PostgreSQL.svg";
+
+const PostgresDataSourceNode = (({data}) => {
```

Comment:
We shouldn't create components for each data source type as they look the same for BigQuery, Postgres, or Snowflake. This should be a "dumb" component.

Also, we should call them `components/blocks/DataBlock.js`